udata/core/followers/models.py

```diff
@@ -15,16 +15,16 @@ class FollowQuerySet(db.BaseQuerySet):
     def following(self, user):
         return self(follower=user, until=None)
 
-    def followers(self, user):
-        return self(following=user, until=None)
+    def followers(self, followee):
```

Comment:
I don't mind having `user` here as it represent explicitely the type of object that is expected.